sub/source/docs/gcs-failsafe.rst

```diff
@@ -0,0 +1,37 @@
+.. _gcs-failsafe:
+
+============
+GCS Failsafe
+============
+The Ground Station Control (GCS) failsafe controls how Sub will behave if contact with the GCS is lost.  The GCS failsafe monitors the time since the last MAVLink heartbeat from the GCS.  If no heartbeat is received :ref:`FS_GCS_TIMEOUT<FS_GCS_TIMEOUT>` seconds (Default is 5 seconds), the GCS failsafe event will trigger based on your parameter settings. Note that if no GCS is ever connected, the GCS failsafe will remain inactive regardless of parameter settings.
+
+.. note::
+
+   Sub also supports other failsafes: :ref:`Radio <radio-failsafe>`, :ref:`Battery <failsafe-battery>`, and :ref:`EKF / DCM Check <ekf-inav-failsafe>`.
```

Comment:
This list seems somewhat random. Should `Pilot Input` perhaps be included instead of `Radio`?

Reply:
changed throughout